.github/workflows/scripts/build.sh

```diff
@@ -15,5 +15,6 @@ $python_executable -m pip install -r requirements-cuda.txt
 export MAX_JOBS=1
 # Make sure release wheels are built for the following architectures
 export TORCH_CUDA_ARCH_LIST="7.0 7.5 8.0 8.6 8.9 9.0+PTX"
+export VLLM_FA_CMAKE_GPU_ARCHES="80-real;90-real"
```

Comment:
What is the "real" for? Does this cover PTX for 8.6/8.9?

Reply:
No I think we specifically decided no ptx, that's how it was build before. The reason this is in a different format is it doesn't go through the torch parsing, but gets used directly by CMake. If ptx was needed, we'd say 80-virtual here.